pkg/operator/ceph/cluster/controller.go

```diff
@@ -130,6 +131,11 @@ func add(opManagerContext context.Context, mgr manager.Manager, r reconcile.Reco
 	}
 	logger.Info("successfully started")
 
+	err = addonsv1alpha1.AddToScheme(mgr.GetScheme())
+	if err != nil {
+		logger.Errorf("failed to add networkFence to scheme: %v", err)
```

Comment:
This means that network fencing cannot be enabled, right? Seems like this should just be a warning unless we know the fencing is expected to be enabled, and then we should fail the reconcile in that case.

Reply:
I think its's better to have it as warning so that it doesn't block current reconcile